unix/vncserver/HOWTO.md

```diff
@@ -74,12 +75,12 @@ You need to run it as the user who will run the server.
 
 ### Note:
 If you used TigerVNC before with your user and you already created a
-password, then you have to make sure the `$HOME/.vnc` folder created by
-`vncpasswd` have the correct *SELinux* context. You either can delete
-this folder and recreate it again by creating the password one more
-time, or alternatively you can run:
+password, then you have to make sure the `$HOME/.config/tigervnc` folder
+created by `vncpasswd` have the correct *SELinux* context. You either
+can delete this folder and recreate it again by creating the password one
+moretime, or alternatively you can run:
 ```
-$ restorecon -RFv /home/<USER>/.vnc
+$ restorecon -RFv /home/<USER>/.config/tigervnc
 ```
```

Comment:
This should probably be left alone. It's only for those upgrading from something before we had an SELinux module. Those using the new paths should be fine.